torch/csrc/jit/passes/onnx/helper.cpp

```diff
@@ -96,5 +96,28 @@ Value* addInputToBlock(Block* block) {
   return block->addInput();
 }
 
+Node* createONNXUnsqueeze(
+    std::shared_ptr<Graph> graph,
```

Comment:
Maybe change this to Graph* to resolve the build failure.

Reply:
thanks for catching, updated.